### ISSUE TITLE
Consider cleaning up temporary directories

### ISSUE THREAD

**Issue by MartinFlores751** [irods/irods_testing_environment#204]
When running tests with less than ideal directory structures, you can end up with temp files that are massive (~19G is what I had), and are left to be cleaned up manually.

It would be nice to have tests cleanup after themselves.

**Comment by trel** [comment]
Consider the balance of *wanting* to see the temp files for forensics...

Perhaps a new knob, with a good default...

**Comment by MartinFlores751** [comment]
> Consider the balance of _wanting_ to see the temp files for forensics...
> 
> Perhaps a new knob, with a good default...

Yes, temp files are nice for that, especially when you forget to copy the logs. Are there any other uses to consider?
